en/docs/get-started/integration-quick-start-guide.md

```diff
@@ -2,25 +2,30 @@
 
 Let's get started with WSO2 Micro Integrator by running a simple integration use case in your local environment. 
 
-## Before you begin
+### Before you begin...
 
-1. Go to the [WSO2 Micro Integrator web page](https://wso2.com/integration/micro-integrator/#), click **Download**, and then click **Zip Archive** to download the Micro Integrator distribution as a ZIP file.
-2. Go to the [API Manager Tooling web page](https://wso2.com/api-management/tooling/), and download WSO2 Integration Studio.
+1. Install [Oracle Java SE Development Kit (JDK)](http://java.sun.com/javase/downloads/index.jsp) version 11 or 1.8 and set the `JAVA_HOME` environment variable.
+   For more information on setting the `JAVA_HOME` environment variable for different operating systems, see [Setup and Install]({{base_path}}/install-and-setup/install/installing-the-product/installing-api-m-runtime/).
+2. Go to the [WSO2 Micro Integrator web page](https://wso2.com/integration/micro-integrator/#), click **Download**, and then click **Zip Archive** to download the Micro Integrator distribution as a ZIP file.
+3. Optionally, Go to the [API Manager Tooling web page](https://wso2.com/api-management/tooling/), and download WSO2 Integration Studio.
```

Comment:
```suggestion
3. Optionally, navigate to the [API Manager Tooling web page](https://wso2.com/api-management/tooling/), and download WSO2 Integration Studio.
```

Reply:
Why is this optional? Don't you need Integration Studio for this?

Reply:
Integration Studio is needed to create the Integration. But in QSG we have provided an already created and tested solution. IS is needed only If one needs to observe the project.